### PR TITLE
Update schema split: `train`, `model`

### PR DESCRIPTION
**Pull request by CBroz1**
See corresponding [workflow PR](https://github.com/CBroz1/workflow-deeplabcut/pull/2). 

Has:
- Updated 2-schema split
- Updated documentation

Recent edits to functions:
- **helper funcs**: fixed bug where root_dir[0] would get appended to root_dirs whenever get_processed_dir was called.
- **extract_new_bodyparts**: added verbose arg to prevent duplicate print when calling model.Model.insert_new_model
- **insert_new_model**: because `BodyPart.extract_new_body_parts` returns a list, checking the truth value is ambiguous. Changed to 'is not None'
- **infer_ouput_dir**: remove _linking_module for VideoRecording table
- **insert_new_params**: Enforce int type on paramset_idx input. Previously failed to return when attempting to insert a duplicate bc str(x)!=int(x)

### CODE REVIEW

File: element_deeplabcut/model.py [hunk 1]
```diff
@@ -529,14 +343,11 @@ def make(self, key):
...
-
-
 @schema
-class PoseEstimationTask(dj.Manual):
+class EstimationTask(dj.Manual):
     definition = """
-    -> VideoRecording
-    -> Model
+    -> VideoRecording                           # Session -> Recording + File part table
+    -> model.Model                              # Must specify a DLC project_path
```
Comment (ttngu207) [inline element_deeplabcut/model.py]:
```suggestion
    -> Model                              # Must specify a DLC project_path

```

File: element_deeplabcut/model.py [hunk 2]
```diff
@@ -529,14 +343,11 @@ def make(self, key):
                           test_error_p=results['Test error with p-cutoff']))
 
 
-# ---- Model Inference pipeline ----
-
-
 @schema
-class PoseEstimationTask(dj.Manual):
+class EstimationTask(dj.Manual):
```
Comment (ttngu207) [inline element_deeplabcut/model.py]:
```suggestion
class PoseEstimationTask(dj.Manual):

```
I'd still prefer the name `PoseEstimationTask` for clarity

File: element_deeplabcut/model.py [hunk 3]
```diff
@@ -610,18 +422,22 @@ class BodyPartPosition(dj.Part):
...
 
         # ID model and directories
-        dlc_model = (Model & key).fetch1()
+        dlc_model = (_linking_module.model.Model & key).fetch1()
+        assert dlc_model['project_path'], ("Your model table must have a 'project_path'"
+                                           + "field pointing to a DLC directory")
         task_mode, analyze_video_params, output_dir = (
-            PoseEstimationTask & key).fetch1('task_mode', 'pose_estimation_params',
-                                             'pose_estimation_output_dir')
+            EstimationTask & key).fetch1('task_mode', 'pose_estimation_params',
```
Comment (ttngu207) [inline element_deeplabcut/model.py]:
```suggestion
            PoseEstimationTask & key).fetch1('task_mode', 'pose_estimation_params',

```

File: element_deeplabcut/model.py [hunk 4]
```diff
@@ -600,7 +434,7 @@ class PoseEstimation(dj.Computed):
     class BodyPartPosition(dj.Part):
         definition = """ # uses DeepLabCut h5 output for body part position
         -> master
-        -> BodyPart
+        -> model.BodyPart
```
Comment (ttngu207) [inline element_deeplabcut/model.py]:
```suggestion
        -> Model.BodyPart

```

File: element_deeplabcut/model.py [hunk 5]
```diff
@@ -610,22 +444,26 @@ class BodyPartPosition(dj.Part):
...
 
     def make(self, key):
-        """.populate() method will launch training for each PoseEstimationTask"""
+        """.populate() method will launch training for each PoseEstimationTask
+        """
         from .readers import dlc_reader
 
         # ID model and directories
-        dlc_model = (Model & key).fetch1()
+        dlc_model = (_linking_module.model.Model & key).fetch1()
```
Comment (ttngu207) [inline element_deeplabcut/model.py]:
```suggestion
        dlc_model = (Model & key).fetch1()

```
Why the need to access `Model` via `_linking_module` and not just directly?

File: element_deeplabcut/model.py [hunk 6]
```diff
@@ -649,7 +465,7 @@ def make(self, key):
     def get_trajectory(cls, key, body_parts='all'):
         """
         Returns a pandas dataframe of x, y and z coordinates of the specified body_parts
-        :param key: A query specifying one PoseEstimation entry, else error is thrown.
+        :param key: A query specifying one Estimation entry, else error is thrown.
```
Comment (CBroz1) [inline element_deeplabcut/model.py]:
```suggestion
        :param key: A query specifying one PoseEstimation entry, else error is thrown.
```

File: element_deeplabcut/model.py (hunk 2 above)
Comment (CBroz1) [reply element_deeplabcut/model.py]: Not sure why you were seeing an outdated version here after [this commit yesterday](https://github.com/CBroz1/element-deeplabcut/pull/5/commits/b84254785e42d2ae07e344bbf11bb84e2979ee51) reverted to `PoseEstimation`

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**